### PR TITLE
Integration tests: Support slower computers/network

### PR DESCRIPTION
**Pull request by neuroscr**
- tweak some timeouts
- better process killing on non-windows platforms
- setValueWrapper to deal with electron/spectron bugs with addValue/SetValue/Keys stalling out (requires #1043 to work)
- add a prefix to all tests
- prefix allows us to run the tests in batches which seems to have fewer false-positives: `yarn test-integration-parts`
- refactored some of the Open Group tests to reuse the same code, so it doesn't get out of sync with each other
- include byId xpath utility in `common.page`

### CODE REVIEW

File: integration_test/open_group_test.js [hunk 1]
```diff
@@ -23,25 +23,25 @@ describe('Open groups', function() {
...
     await app.client.element(ConversationPage.joinOpenGroupButton).click();
 
     // validate session loader is shown
     await app.client.isExisting(ConversationPage.sessionLoader).should
       .eventually.be.true;
+    // account for slow home internet connection delays...
     await app.client.waitForExist(
       ConversationPage.sessionToastJoinOpenGroupSuccess,
-      9000
+      60 * 1000
```
Comment (Bilb) [inline integration_test/open_group_test.js:44]: the maximum test timeout is currently 30seconds. you want to increase that value too, isn't it? https://github.com/loki-project/session-desktop/pull/1044/files?file-filters%5B%5D=.js#diff-dd50b247dac58a9b47120e5c4f1b2c46R9

File: integration_test/open_group_test.js [hunk 2]
```diff
@@ -23,25 +23,25 @@ describe('Open groups', function() {
     await common.killallElectron();
   });
 
-  it('works with valid group url', async () => {
+  // reduce code duplication to get the initial join
+  async function joinOpenGroup(url, name) {
```
Comment (Bilb) [inline integration_test/open_group_test.js:27]: could be in `common.js`

File: integration_test/open_group_test.js [hunk 3]
```diff
@@ -50,64 +50,64 @@ describe('Open groups', function() {
...
+      common.VALID_GROUP_URL2
+    );
     await app.client.element(ConversationPage.joinOpenGroupButton).click();
-    // first add is a success
-    await common.timeout(2000);
+
+    // wait for toast to appear
+    await app.client.waitForExist(
+      ConversationPage.sessionToastJoinOpenGroupSuccess,
+      30 * 1000
```
Comment (Bilb) [inline integration_test/open_group_test.js:104]: same as above. current test timeout is 30s https://github.com/loki-project/session-desktop/pull/1044/files?file-filters%5B%5D=.js#diff-dd50b247dac58a9b47120e5c4f1b2c46R9

File: integration_test/open_group_test.js (hunk 2 above)
Comment (neuroscr) [reply integration_test/open_group_test.js:27]: no, I feel like common shouldn't have functions for specific tests. In fact I'd favor moving some of the constants out of it. I think we should favor smaller cleaner files (even though, I personally have no problem working with bigger ones).

File: integration_test/open_group_test.js (hunk 1 above)
Comment (neuroscr) [reply integration_test/open_group_test.js:44]: we can reduce these down, I didn't need that high but I could see other running the integration tests (maybe outside devs) may need it. 

I'm fine either way tbh.

**Pull request merged**